pipeline/terraform0.12/modules/pipeline_service/variables.tf

```diff
@@ -0,0 +1,50 @@
+variable "service_name" {}
+variable "cluster_arn" {}
+variable "cluster_name" {}
+
+variable "subnets" {
+  type = list(string)
+}
+
+variable "aws_region" {}
+variable "namespace_id" {}
+variable "container_image" {}
+
+variable "secret_env_vars" {
+  type = map(string)
+}
+
+variable "env_vars" {
+  type = map(string)
+}
+
+variable "security_group_ids" {
+  default = []
+  type = list(string)
+}
+
+variable "max_capacity" {}
+
+variable "messages_bucket_arn" {}
+
+variable "queue_read_policy" {}
+
+variable "desired_task_count" {
+  default = 1
+}
+
+variable "launch_type" {
+  default = "FARGATE"
+}
+
+variable "cpu" {
+  default = "256"
+}
+
+variable "memory" {
+  default = "512"
```

Comment:
Suggestion: add `type = number` to these variables, and `desired_task_count` above

Reply:
Nice suggestion - done, thanks